src/js/core.js

```diff
@@ -339,17 +339,14 @@
             element.setAttribute('role', 'textbox');
             element.setAttribute('aria-multiline', true);
             element.setAttribute('medium-editor-index', index);
+        }, this);
 
-            if (element.hasAttribute('medium-editor-textarea-id')) {
```

Comment:
Yeah, I understand. But previously, ```input``` event was attached to element itself. But now ```editableInput``` is attached to the editor, not element. Maybe I could add a condition, if there is at least one element with textarea, then listen for ```editableInput```.